_doctest_environment.py

```diff
@@ -0,0 +1,4 @@
+# Toplevel for doctesting with passagemath
+
+from sage.all__sagemath_modules import *
+from sage.all__sagemath_symbolics import *
```

Comment:
Ruff fails on this because of `import *`. If this is necessary for the doctests you'd need to add
```suggestion
from sage.all__sagemath_modules import *  # noqa: F403
from sage.all__sagemath_symbolics import *  # noqa: F403
```